packages/features/eventtypes/components/tabs/instant/InstantEventController.tsx

```diff
@@ -171,6 +176,44 @@ export default function InstantEventController({
                             );
                           }}
                         />
+                        <div>
+                          <Label>{t("only_show_if_parameter_set")}</Label>
+                          <div className="space-y-2">
```

Comment:
In the next iteration i would be refactoring it to something like this

![Screenshot 2024-11-15 at 5 47 26 PM](https://github.com/user-attachments/assets/03cbb50b-fb28-40a7-9111-731edb0a2210)

Reply:
https://github.com/calcom/cal.com/pull/17361 Something similar is achieved here.